nupic/algorithms/anomaly.py

```diff
@@ -84,22 +93,15 @@ def __init__(self, slidingWindowSize=None, mode=MODE_PURE, binaryAnomalyThreshol
               models probability of receiving this value and anomalyScore
           - "weighted" - "pure" anomaly weighted by "likelihood"
               (anomaly * likelihood)
+          - OR type function and will be used directly to compute score
```

Comment:
I agree, decoupled the `mode`(now string only) and callable `customComputeFn`. A disadvantage is that now CLAModel has to be updated to "read" customComputeFn param from OPF description files.